### ISSUE TITLE
voiceover cannot focus on the Show Full Message button

### ISSUE THREAD

**Issue by fastfinge** [deltachat/deltachat-ios#3270]
- iOS version: IOS 26, IOS 27
- Device: iPhone 16 pro max
- Delta Chat version: latest
- Expected behavior: Voiceover should be able to swipe to and press the Show Full Message… button.
- Actual behavior: After getting a lengthy message, voiceover on IOS cannot press Show Full Message… and read the text of the message.
- Steps to reproduce the problem: Turn on voiceover. Get a lengthy message. Try to press show full message and read the message text.
- Screenshots:
- Logs:
